apps/yggdrasil/lib/yggdrasil/user.ex

```diff
@@ -29,14 +33,46 @@ defmodule Yggdrasil.User do
   end
 
   @doc """
-  adds password at end of chain protects the hashpwsalt from seeing a nil value
-  in the case of missing password field.
+  Creates the user and assigns the default roles of "player"
   """
-  def hash_password(changeset = %{:valid? => false}) do
+  def create_with_default_role(attributes) do
```

Comment:
Stashing this here for now. I can't do this in the change set it self, because the user must first be inserted into the db.